src/http_server/api/v1/health.c

```diff
@@ -323,7 +323,7 @@ int api_v1_health(struct flb_hs *hs)
 
     counter_init(hs);
     /* Create a message queue */
-    hs->qid_metrics = mk_mq_create(hs->ctx, "/health",
+    hs->qid_health = mk_mq_create(hs->ctx, "/health",
```

Comment:
So previously health check was consuming from the normal metrics queue, now we duplicate the data to a new queue? I feel like there should be more changes to ensure the health check handler reads from the new queue?

@DrewZhang13

Reply:
Note: The new queue (qid_health) was already defined at original PR (#3494) and it has not been used.

https://github.com/fluent/fluent-bit/pull/3494/files#diff-3978152c3bf1385d1b2e87e5fbf941f4bdf14ddac95bdb70436789d2c1b0d876R46
https://github.com/fluent/fluent-bit/pull/3494/files#diff-9fadfecb6c5976e7f2c6fa4abc77a4e78329941efb5905a5d38d179730993058R43

Reply:
This PR added the method to push metrics to the new queue above at line 174. With that one, the metrics message will be pushed to both metrics queue and health queue. 
Current line is registering the new queuefor health check to read. 
Overall the logic looks correct to me.